bus-mall.js

```diff
@@ -0,0 +1,109 @@
+// refresh page, .random choseing imgags, images stored in array, <15, create counter/
+//whileloop. event listner < on click refresh page
+
+
+var productNames = ['bag', 'banana', 'bathroom', 'boots', 'breakfast', 'bubblegum', 'chair', 'cthulhu', 'dog-duck', 'dragon', 'pen', 'pet-sweep', 'scissors', 'shark', 'tauntaun', 'unicorn', 'water-can', 'wine-glass'];
+
+var productArr = [];
+
+// Constructor function
+function Product(name) {
+ this.name = name;
+ this.votes = 0;
+ this.path = 'image/' + name + '.jpg';
+ productArr.push(this);
+}
+console.log(this.path);
+// a simple IIFE to build all the product images // Producing path to new product
+(function () {
+ for (var i = 0; i < productNames.length; i++) {
+   new Product(productNames[i]);
+ }
+ console.log(productArr);
+})();
+
+var tracker = {
+  imageOneEl: document.getElementById('imageOne'),
```

Comment:
When you have three identical things that you want to iterate through, what do you put them in?